app/src/main/java/com/keylesspalace/tusky/service/SendStatusService.kt

```diff
@@ -154,56 +151,56 @@ class SendStatusService : Service(), Injectable {
 
         val callback = object : Callback<Status> {
```

Comment:
yikes, it's still super old network API, should we also replace it? Might make the rest much easier to read, idk